codalab/worker/worker.py

```diff
@@ -622,6 +625,15 @@ def initialize_run(self, bundle, resources):
                 bundle=bundle,
                 bundle_path=os.path.realpath(bundle_path),
                 bundle_dir_wait_num_tries=Worker.BUNDLE_DIR_WAIT_NUM_TRIES,
+                bundle_profile_stats={
+                    RunStage.PREPARING: self.initstats(),
```

Comment:
If we store all this information, why are we only storing times in metadata for preparing, running, cleaning up, and uploading results?

Reply:
I should probably remove the extraneous stuff.

it goes from uploading results->finalizing, and that seems to be the last stage (there is a start time set but no end time). I could not find where it gets out of finalizing, maybe if I set the end time there we could have the finalizing stage.